app/views/api_docs/pages/release_notes.md

```diff
@@ -1,6 +1,12 @@
 ## 25th February
+New attributes:
+
+- `Candidate` now has a `fee_payer` attribute of type string, returning a two-digit string corresponding to UCAS Fee Payer codes. It indicates a provisional fee payer status. Its value is derived from the candidate's nationality, right to work or study in the UK and domicile.
+
+Documentation:
 
-- Documentation updated to indicate that the `Candidate.domicile` field is encoded as a HESA DOMICILE code.
+- Clarify the description of `Candidate.uk_residency_status` field in the documentation.
+- Indicate that the `Candidate.domicile` field is encoded as a HESA DOMICILE code.
```

Comment:
This went in with #4162

Reply:
I'm hoping to still deploy it today. That's why I merged those under todays date.

Reply:
OK great